datasets/ky/parliament/ky_parliament.yml

```diff
@@ -26,3 +26,33 @@ publisher:
 data:
     url: https://parliament.ky/members/
     format: HTML
+
+assertions:
+  - comparison: GT
+    threshold: 100
+    action: WARN
+    filter:
+      attribute: schema
+      value: Person
```

Comment:
to support totally different metrics like number of countries, we could rather do

```yaml
assertions:
  - metric: entity_count
    comparison: GT
    threshold: 100
    action: WARN
    filter:
      attribute: schema
      value: Person

  - metric: country_count
    comparison: GT
    threshold: 123
    action: WARN
```